packages/x-adapter/src/schemas/utils.ts

```diff
@@ -0,0 +1,93 @@
+import { deepMerge } from '@empathyco/x-deep-merge';
+import { MutableSchema, Schema } from './schemas.types';
+
+/**
+ * Collection of internal method names for {@link MutableSchema | mutable schemas}.
+ */
+const mutableSchemasInternalMethods = ['$replace', '$override', '$extends', '$toString'];
+
+/**
+ * Creates a {@link MutableSchema | mutable schema } version of a given {@link Schema | schema}.
+ *
+ * @param schema - The {@link Schema | schema} to make mutable.
+ *
+ * @returns A {@link MutableSchema | mutable schema} version of the given {@link Schema | schema}.
+ *
+ * @public
+ */
+export function createMutableSchema<T extends Schema>(schema: T): MutableSchema<T> {
+  return {
+    ...schema,
+    $replace: function <Source = any, Target = any>(
+      newSchema: Schema<Source, Target>
+    ): MutableSchema<Schema<Source, Target>> {
+      Object.keys(this).forEach(key => {
+        if (isInternalMethod(key)) {
+          return;
+        }
+        delete this[key];
+      });
+      Object.assign(this, newSchema);
+      return this;
+    },
+    $override: function <Source = any, Target = any>(
+      newSchema: Schema<Source, Target>
+    ): MutableSchema<Schema<Source, Target>> {
+      return deepMerge(this, newSchema);
+    },
+    $extends: function <Source = any, Target = any>(
+      newSchema: Schema<Source, Target>
+    ): MutableSchema<Schema<Source, Target>> {
+      return deepMerge({}, this, newSchema);
+    },
+    $toString: function (includeInternalMethods = false) {
```

Comment:
rename to standard `toString` to coercing directly to a string:

`console.log(schema)`